.github/workflows/build.yml

```diff
@@ -166,21 +173,3 @@ jobs:
           if-no-files-found: ignore
       - if: steps.changes.outputs.changed == 1
         run: echo "Working directory is dirty" && exit 1
-
-  build-windows:
-    runs-on: windows-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: setup msys2
-        uses: msys2/setup-msys2@v2
-        with:
-          msystem: MINGW64
-          update: false
-          install: git msys2-devel base-devel mingw-w64-x86_64-gcc mingw-w64-x86_64-go  mingw-w64-x86_64-jq  mingw-w64-x86_64-dlfcn
-      - name: get rocksdb & build
-        env:
-          PLATFORM: Windows_x86_64
-        shell: msys2 {0}
-        run: |
-          ./scripts/build-windows.sh
```

Comment:
might remove the [script](https://github.com/crypto-org-chain/cronos/blob/4939ae4bac754f3818582c221cc5b0400cc99028/scripts/build-windows.sh) as well

Reply:
done